.github/workflows/test-suite-desktop-e2e.yml

```
@@ -36,39 +36,75 @@ jobs:
+          # Fetch previous PR runs
+          PREVIOUS_RUN_JOBS_URL=$(curl -s -H "Authorization: Bearer ${GITHUB_TOKEN}" "https://api.github.com/repos/${REPO}/actions/workflows/${WORKFLOW_ID}/runs?event=pull_request&branch=${PR_BRANCH}&status=completed&per_page=1" | jq -r '.workflow_runs[] | .jobs_url')
+
+          if [ -z "$PREVIOUS_RUN_JOBS_URL" ]; then
+            echo "No previous runs found. Running tests as usual"
+            exit 0
+          fi
+
+          echo ${{ github.job }}
+
+          SUCCESSFUL_JOB=$(curl -s -H "Authorization: Bearer ${GITHUB_TOKEN}" https://api.github.com/repos/trezor/trezor-suite/actions/runs/13386553182/jobs | jq -r --arg JOB_NAME ${{ github.job }}" '.jobs[] | select(.name == $JOB_NAME and .conclusion == "success") | .url')
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)